plugins/torrent/deluge_

```diff
@@ -207,10 +222,13 @@ class StatClient:
                 self.end_session,
                 errbackArgs=("get_session_state failed"))
```

Comment:
Please add an `else` branch here (and around line 378) and emit an error message in case of an unknown plugin view.
This is specifically relevant, since the update removes a previously working view ("connections"). Users of the previous plugin should get a fair chance to notice the change.

Reply:
I did that now. 

It's not really required with the current code flow since the plugin confirms the plugin view via the `get_mode` function. I still added the `else` clause to solve your comment. Let me know if you think it's overkill given this new information and I can remove it again.

```
def get_mode():
    script_name = os.path.basename(sys.argv[0])
    mode = script_name[script_name.rindex('_') + 1:]

    log.debug("Mode : %s", mode)

    if mode not in modes:
        log.error("Unknown mode '%s'", mode)
        log.info("Available modes are : %s", modes)
        sys.exit(1)

    return mode
```

Reply:
Thank you for committing to my style obsessions :)

Personally I always feel the urge to verify such checks, whenever I stumble upon a non-trivial `if/elif` flow. Adding the `else` clause takes this burden away from me (and maybe others). This somehow follows the thought of "code is read much more often than it is written" ...